easybuild/easyconfigs/g/GROMACS/GROMACS-4.6.7-CrayGNU-2015.11-XC-mpi.eb

```diff
@@ -0,0 +1,45 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+##
+name = 'GROMACS'
+version = '4.6.7'
+versionsuffix = '-mpi'
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+toolchainopts = {'usempi': True}
+
+# eg. ftp://ftp.gromacs.org/pub/gromacs/gromacs-4.6.tar.gz
+source_urls = [
+    'ftp://ftp.gromacs.org/pub/gromacs/',  # GROMACS sources
+    'http://gerrit.gromacs.org/download/',  # regression tests sources
+]
+sources = [
+    SOURCELOWER_TAR_GZ,
+    'regressiontests-%(version)s.tar.gz',
+]
+
+preconfigopts  = "export CMAKE_LIBRARY_PATH=$CMAKE_LIBRARY_PATH:${EBROOTFFTW} && "
```

Comment:
BTW, this made me realise that we should really get https://github.com/hpcugent/easybuild-framework/pull/1586 merged in before we declare Cray support stable, since getting the metadata for the Cray modules right is very important to achieve success.

In fact, we were rather lucky to stumble into this. If you would've used `FFTW_DIR` like @gppezzi has been using in his metadata file, I wouldn't have noticed this issue...